### PR TITLE
Cisco Phones HTTP provisioning

### PR DESCRIPTION
**Pull request by vincentpeyrouse**
Added support for Cisco phones HTTP provisioning (starting 9.3.x firmwares).

### CODE REVIEW

File: plugins/xivo-cisco-sccp/common/common.py [hunk 1]
```diff
@@ -209,10 +237,17 @@ def __init__(self, app, plugin_dir, gen_cfg, spec_cfg):
         fetchfw_helper = FetchfwPluginHelper(plugin_dir, downloaders)
 
         self.services = fetchfw_helper.services()
+
+        # Maybe find a way to bind to a specific port without changing the general http_port setting of xivo-provd ?
```
Comment (sduthil) [inline plugins/xivo-cisco-sccp/common/common.py]: Another solution would be to add some reverse-proxy configuration in Nginx to redirect HTTP port 6970 to port 8667.

Comment (sboily) [reply plugins/xivo-cisco-sccp/common/common.py]: There is no solution to send via DHCP another port for the provisioning?
Thank you for the PR.

Comment (vincentpeyrouse) [reply plugins/xivo-cisco-sccp/common/common.py]: Well i'm not pretty sure, but it seems not :(
I found nothing like that in Cisco docs whether firmwares or call manager updates releases notes.

**Comment by sduthil** [comment]
Merged and uploaded to the stable provisioning repo. Thank you very much! :)

**Comment by sduthil** [comment]
I added issue https://projects.wazo.community/issues/7162 to explain the configuration needed for HTTP provisioning.

**Pull request merged**